### PR TITLE
feat: link socat to /usr/local/bin

### PR DESCRIPTION
**Pull request by maxmoehl**

### CODE REVIEW

**Review by peanball — APPROVED** [review]

File: jobs/haproxy/templates/pre-start.erb [hunk 1]
```diff
@@ -16,3 +16,7 @@ fi
 if [ ! -e /usr/local/bin/hatop ]; then
   sudo ln -s /var/vcap/packages/haproxy/hatop-wrapper /usr/local/bin/hatop
 fi
+
+if [ ! -e /usr/local/bin/socat ]; then
+  sudo ln -s /var/vcap/packages/haproxy/socat /usr/local/bin/socat
```
Comment (plowin) [inline jobs/haproxy/templates/pre-start.erb:21]: Hi @maxmoehl , only `/var/vcap/packages/haproxy/bin/socat` exists, this works for me:
```
sudo ln -s /var/vcap/packages/haproxy/bin/socat /usr/local/bin/socat
```

The symlink creation of this PR does not fail but points into nowhere. The acceptance tests could be more expressive but I think that they are catching such an issue at all is already good enough :)

**Review by plowin — CHANGES_REQUESTED** [review]

File: jobs/haproxy/templates/pre-start.erb (hunk 1 above)
Comment (maxmoehl) [reply jobs/haproxy/templates/pre-start.erb:21]: 🤦🏻‍♂️ Thanks!

Comment (maxmoehl) [reply jobs/haproxy/templates/pre-start.erb:21]: Classic Copy-Paste issue 😄

**Review by plowin — APPROVED** [review]
Thx, lgtm!
I added intial release-notes, please cross-check/comment them before merging. Other features can be added on-top as we will not cut a release directly

**Pull request merged**